src/main/java/com/devcycle/sdk/server/local/model/Project.java

```diff
@@ -1,5 +1,7 @@
 package com.devcycle.sdk.server.local.model;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
 public class Project {
```

Comment:
do we need to add the annotation of `@JsonIgnoreProperties(ignoreUnknown = true)` on the ProjectSettings class?

Reply:
ah.. because this class isn't used in a builder pattern.. that's why no impact on fixing the failed JSON parsing